virtualpdu/core.py

```diff
@@ -27,11 +27,13 @@ def __init__(self, driver, mapping):
     def pdu_outlet_state_changed(self, name, outlet_number, state):
         self.logger.info(
             "PDU '{name}', outlet '{outlet_number}' has new state: '{state}'".
-            format(name=name, outlet_number=outlet_number, state=state))
+                format(name=name, outlet_number=outlet_number, state=state))
         try:
             server_name = self._get_server_name(name, outlet_number)
 
-            self.logger.info("Found server '{0}'".format(server_name))
+            self.logger.debug("Found server '{0}' on PDU '{1}', "
```

Comment:
line 29 uses named params, and this uses ordinal params, you guys should stick to 1 standard :)  i suggest {} :P